salt/reactors/edx/pulsar_mitx.sls

```diff
@@ -0,0 +1,8 @@
+pulsar_mitx:
+  local.slack.post_message:
+    - tgt: {{ data['id'] }}
```

Comment:
If u do cmd.slack.post_message: ... Would that run locally or is that running on the minion?